tracim/development.ini.base

```diff
@@ -34,8 +34,13 @@ use = egg:tracim
 full_stack = true
 # You can set french as default language by uncommenting next line
 # lang = fr
-cache_dir = %(here)s/data
```

Comment:
I believe this parameter is a turbogears core parameter. It is probably a bad idea to re-use it four our use except if you know exactly what you are doing. You may broke something without to know it.